renderdoc/driver/vulkan/vk_common.h

```diff
@@ -945,6 +945,37 @@ enum class VulkanChunk : uint32_t
   vkCmdSetFragmentShadingRateKHR,
   vkSetDeviceMemoryPriorityEXT,
   vkCmdSetAttachmentFeedbackLoopEnableEXT,
+  vkCmdSetAlphaToCoverageEnableEXT,
+  vkCmdSetAlphaToOneEnableEXT,
+  vkCmdSetColorBlendAdvancedEXT,
+  vkCmdSetColorBlendEnableEXT,
+  vkCmdSetColorBlendEquationEXT,
+  vkCmdSetColorWriteMaskEXT,
+  vkCmdSetConservativeRasterizationModeEXT,
+  vkCmdSetCoverageModulationModeNV,
+  vkCmdSetCoverageModulationTableEnableNV,
+  vkCmdSetCoverageModulationTableNV,
+  vkCmdSetCoverageReductionModeNV,
+  vkCmdSetCoverageToColorEnableNV,
+  vkCmdSetCoverageToColorLocationNV,
+  vkCmdSetDepthClampEnableEXT,
+  vkCmdSetDepthClipEnableEXT,
+  vkCmdSetDepthClipNegativeOneToOneEXT,
+  vkCmdSetExtraPrimitiveOverestimationSizeEXT,
+  vkCmdSetLineRasterizationModeEXT,
+  vkCmdSetLineStippleEnableEXT,
+  vkCmdSetLogicOpEnableEXT,
+  vkCmdSetPolygonModeEXT,
+  vkCmdSetProvokingVertexModeEXT,
+  vkCmdSetRasterizationSamplesEXT,
+  vkCmdSetRasterizationStreamEXT,
+  vkCmdSetRepresentativeFragmentTestEnableNV,
+  vkCmdSetSampleLocationsEnableEXT,
+  vkCmdSetSampleMaskEXT,
+  vkCmdSetShadingRateImageEnableNV,
+  vkCmdSetTessellationDomainOriginEXT,
+  vkCmdSetViewportSwizzleNV,
```

Comment:
I'd prefer to entirely omit unnecessary references to entirely unsupported functions. It's fine to have stubs as required, but no serialisation/etc references.